homes-ms-storesmanagement/src/main/java/com/tenx/ms/storesmanagement/stores/rest/dto/StoreDTO.java

```diff
@@ -0,0 +1,75 @@
+package com.tenx.ms.storesmanagement.stores.rest.dto;
+
+
+import com.tenx.ms.commons.validation.constraints.PhoneNumber;
+import io.swagger.annotations.ApiModelProperty;
+import org.apache.avro.reflect.Nullable;
+
+import javax.validation.constraints.NotNull;
+
+
+public class StoreDTO {
+    @ApiModelProperty(value = "Store identificator", readOnly = true)
+    private Long storeId;
+
+    @ApiModelProperty(value = "Store name")
+    @NotNull
+    private String name;
+
+    @ApiModelProperty("Store address")
+    @Nullable
```

Comment:
Just FYI, this Nullable annotation is actually used for Avro schema to tell the deserializer of Kafka message that certain attributes can be null. It's probably not what you intend to use. By default if you don't put NotNull annotation on an attribute, it's nullable automatically without any additional annotation.